patroni/ctl.py

```diff
@@ -46,6 +46,7 @@
 except ImportError:  # pragma: no cover
     from cdiff import markup_to_pager, PatchStream  # pyright: ignore [reportMissingModuleSource]
 
+from .config import get_global_config
```

Comment:
```suggestion
from .config import Config, get_global_config
```

and remove import from the load_config() method.